cmd/kubeadm/app/cmd/join.go

```diff
@@ -31,35 +31,36 @@ import (
 )
 
 var (
-	join_done_msgf = dedent.Dedent(`
+	joinDoneMsgf = dedent.Dedent(`
 		Node join complete:
 		* Certificate signing request sent to master and response
 		  received.
 		* Kubelet informed of new secure connection details.
 
-		Run 'kubectl get nodes' on the master to see this node join.
+		Run 'kubectl get nodes' on the master to see this machine join.
 		`)
 )
 
+// NewCmdJoin returns "kubeadm join" command.
 func NewCmdJoin(out io.Writer, s *kubeadmapi.KubeadmConfig) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "join",
-		Short: "Run this on other servers to join an existing cluster.",
+		Short: "Run this on any machine you wish to join an existing cluster.",
```

Comment:
Please can you add a `Long` description here, which explains what will happen when you run this command?